examples/datasets/workflow.js

```diff
@@ -0,0 +1,14 @@
+import 'dotenv/config';
+import { getDatasets, uploadDataset } from '@rungalileo/galileo';
+
+// Upload a dataset
+const dataset = await uploadDataset({
+  col1: ['val1', 'val2'],
+  col2: ['val3', 'val4']
+});
+console.log(`Uploaded datset: ${dataset.name}`);
+
+// Get all datasets
+console.log('All datasets:');
+const datasets = await getDatasets();
+datasets.forEach((dataset) => console.log(dataset));
```

Comment:
Can we make this a unit test instead?

Reply:
I'd really like to autogenerate our api client from the openapi spec and swapping out our custom api clients before writing unit tests.

Reply:
I think that's a separate setup, which we consider / do independently. But we should atleast have some simple tests for flows we expect to validate for.

Reply:
I think we should just do it now, it's a fairly simple change, starting with https://github.com/rungalileo/galileo-js/pull/37